_translator-files/po/es/QOS-Windows.po

```diff
@@ -3,73 +3,74 @@
 msgid "{% include breadcrumb.html root=\"More\" %}"
-msgstr ""
+msgstr "{% include breadcrumb.html root=\"Más\" %}"
 
 #. type: Plain text
 #: ../wiki/en/QOS-Windows.md:11
 msgid "## Use of QoS on Windows"
-msgstr ""
+msgstr "## Uso de QoS en Windows"
 
 #. type: Plain text
 #: ../wiki/en/QOS-Windows.md:13
 msgid "Jamulus uses Quality of Service (QoS) to mitigate network jitter delays."
-msgstr ""
+msgstr "Jamulus Utiliza Calidad de Servicio (QoS) para mitigar los retardos de red causados por jitter."
 
 #. type: Plain text
 #: ../wiki/en/QOS-Windows.md:15
 msgid "If you want to explore the effect of non-default settings, see [RFC4594](https://datatracker.ietf.org/doc/html/rfc4594) pages 16-18.  Jamulus sets the DS Field byte to 128 (or 0x80) to select DSCP/CS4 in an IPv4 or IPv6 packet header. Other byte values can be set using the -Q option, e.g. -Q [0..255].  However, most people will have no need to do this."
-msgstr ""
+msgstr "Si quieres explorar los efectos de configuraciones no predeterminadas, consulta [RFC4594](https://datatracker.ietf.org/doc/html/rfc4594) páginas 16-18.  Jamulus establece el DS Field byte en 128 (o 0x80) para seleccionar DSCP/CS4 en una cabecera de paquete IPv4 o IPv6. Se pueden establecer otros valores de bytes con la opción -Q, por ej. -Q [0..255].  Sin embargo, la mayoría de las personas no necesitan hacer esto."
 
 #. type: Plain text
 #: ../wiki/en/QOS-Windows.md:17
 msgid "Jamulus' QoS settings (including the default) are set to 0 on recent Windows and macOS because of the operating system. To use QoS on Windows, follow these instructions. Note that you may need to repeat this procedure every time Jamulus is updated."
-msgstr ""
+msgstr "La configuración QoS de Jamulus (incluyendo la predeterminada) se establece en 0 en versiones recientes de Windows y macOS a causa del sistema operativo. Para utilizar QoS en Windows, sigue estas instrucciones. Es posible que necesites repetir este procedimiento cada vez que Jamulus se actualiza."
 
 #. type: Plain text
 #: ../wiki/en/QOS-Windows.md:37
 msgid "In Search box beside Start menu, Type: Local Group Policy Editor (enter)<br> In new window, (click) on the menu icon to display the Actions third panel<br> Looking at the first panel of the Local Group Policy Editor<br> &nbsp;Local Computer Policy<br> &nbsp;&nbsp;Computer Configuration<br> &nbsp;&nbsp;&nbsp;Windows Settings<br> &nbsp;&nbsp;&nbsp;&nbsp;Policy-based QoS (click)<br> Looking at the third panel (Actions) of the Local Group Policy Editor<br> &nbsp;Policy-based QoS<br> &nbsp;&nbsp;More Actions<br> &nbsp;&nbsp;&nbsp;Create new Policy (click)<br> &nbsp;&nbsp;&nbsp;&nbsp;Policy Name: Jamulus<br> &nbsp;&nbsp;&nbsp;&nbsp;Specify DSCP value: 32<br> &nbsp;&nbsp;&nbsp;&nbsp;Next<br> &nbsp;&nbsp;&nbsp;&nbsp;This QoS policy applies Only to applications with name Jamulus.exe<br> &nbsp;&nbsp;&nbsp;&nbsp;Next<br> &nbsp;&nbsp;&nbsp;&nbsp;Next<br> &nbsp;&nbsp;&nbsp;&nbsp;UDP<br> &nbsp;&nbsp;&nbsp;&nbsp;Finish<br>"
```

Comment:
Eek.  Is that meant to turn into a table? Doesn't the MD renderer do tables?

Reply:
No, it turns into an indented list.

Reply:
Oh... well MD definitely has them...  Hm.  Another issue for it, though.